framework/doc/content/source/markers/BoundaryMarker.md

```diff
@@ -0,0 +1,38 @@
+# BoundaryMarker
+
+!syntax description /Adaptivity/Markers/BoundaryMarker
+
+## Description
+
+The `BoundaryMarker` is a stand-alone marker that marks all
+elements either directly or within a given `distance` to a boundary.
+
+!alert warning title=Distributed Mesh
```

Comment:
Should you not put a mooseWarning in the code for this? Non-deterministic behavior in parallel is not so good. That or gather the points for distributed mesh.

Reply:
Yeah, I guess 🙎